registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -63,6 +63,46 @@
           "name": "RMRK"
         }
       ]
+    },
+    "0x9f25cb1fa8328de2fdfcbb16ed3e0d2bb65ee3ce573a8a0d855c6589b96d8d04": {
+      "tokenId": "0x9f25cb1fa8328de2fdfcbb16ed3e0d2bb65ee3ce573a8a0d855c6589b96d8d04",
+      "deployer": "0x4126Bb708458fe7C9126DD7854255F57ac3e1d67",
+      "originalMinter": null,
+      "prettySymbol": "MVX",
+      "decimals": 18,
+      "originAxelarChainId": "polygon",
+      "tokenType": "canonical",
+      "deploySalt": "0x",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/images/tokens/mvx.svg"
```

Comment:
there was a typo in our scripts (which we'll fix). in the meantime, can you change this to `https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/mvx.svg`?